models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php

```diff
@@ -146,40 +112,51 @@ public function createContainer()
     /**
      * @inheritdoc
      */
-    public function add(TaskInterface $task, $status, $label = null)
+    public function add(TaskInterface $task, string $status, ?string $label = null): void
```

Comment:
```suggestion
    public function add(TaskInterface $task, string $status, string $label = null): void
```
Default value already provided